paper_pretrained_models/README.md

```diff
@@ -1,60 +0,0 @@
-# Run DeepRank-GNN pre-trained model on your own data
```

Comment:
I deleted all `paper_pretrained_models` folder, since it should be relative to the previous Deeprank-GNN repo. Just confirm that this is okay to be sure.

Reply:
I never needed them for any test or project. I have no idea why it's there. Perhaps @LilySnow knows.